physics/GWD/drag_suite.meta

```diff
@@ -593,6 +600,14 @@
   dimensions = ()
   type = logical
   intent = in
+[psl_gwd_dx_factor]
+  standard_name = effective_grid_spacing_of_psl_gwd_suite
+  long_name = multiplication of grid spacing
+  units = count
```

Comment:
According to https://github.com/ESCOMP/CCPPStandardNames/blob/main/StandardNamesRules.rst, the units should be `1` not count. Please make the change here and in fv3atm.

Reply:
changed